layouts/partials/footer.html

```diff
@@ -5,37 +5,50 @@
   {{ $orgUrl:= site.Params.brand.parent_org_url }}
   <div class="container">
     <div class="row align-items-center">
-    <div class="col-md-3 text-md-left text-center">
-       <ul>
-        <li><a class="nav-link text-white" href="{{ site.BaseURL | relLangURL }}">HOME</a></li>
-        {{ range site.Menus.main }}
+      <div class="col-md-3 text-md-left text-center">
+        <ul>
+          {{ range site.Menus.main }}
           {{if ne .Name "pages"}}
             <li class="nav-item">
               <a class="nav-link text-white text-uppercase" href="{{ .URL | absLangURL }}">{{ .Name }}</a>
             </li>
+          {{else if .HasChildren }}
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle text-white text-uppercase" href="#" role="button" data-toggle="dropdown"
+                aria-haspopup="true" aria-expanded="false"> {{ .Name }} 
+              </a>
+              <div class="dropdown-menu">
+                {{ range .Children }}
+                  <a class="dropdown-item" href="{{ .URL | absLangURL }}">{{ .Name }}</a>
```

Comment:
Adding the logic into the footer duplicates the same code we have in the [`navigation.html` partial](https://github.com/unicef/inventory-hugo-theme/blob/main/layouts/partials/navigation.html#L34-L43). Instead, could we make a new partial to handle the navigation menu, and embed that new partial into the navigation and footer menus?